themes/tiddlywiki/vanilla/print.tid

```diff
@@ -0,0 +1,195 @@
+title: $:/themes/tiddlywiki/vanilla/print
+tags: $:/tags/Stylesheet
+
+\define set-page-break-tiddlers()
+
+@media print {
+	[data-tiddler-title='$(currentTiddler)$'].tc-tiddler-frame {
+		page-break-before: always;
+	}
+}
+
```

Comment:
do we need those empty lines?

Reply:
No I don't think that we need them, I'll format it better